pandas/core/arrays/base.py

```diff
@@ -708,6 +712,31 @@ def _add_comparison_ops(cls):
         cls.__le__ = cls._create_comparison_method(operator.le)
         cls.__ge__ = cls._create_comparison_method(operator.ge)
 
+    def _reduce(self, op, name, axis=0, skipna=True, numeric_only=None,
+                filter_type=None, **kwargs):
```

Comment:
What are `filter_type` and `kwargs` doing? 
I also think `numeric_only` and `axis` is not relevant for EA?

Is `op` always a numpy function? (and what are the possible values?)

Reply:
From the code below I see that `op` is rather one of our own nanops functions?